src/local-mode.ts

```diff
@@ -162,14 +163,25 @@ export class LocalModeManager {
 			);
 		} catch (e) {
 			if (this.containerId !== undefined) {
-				// Inspect operation fails (container ID is out of sync?).
-				const fallback = SUPERVISOR_CONTAINER_NAME_FALLBACK;
-				log.warn(
-					'Supervisor container resources cannot be obtained by container ID. ' +
-						`Using '${fallback}' name instead.`,
-					e.message,
-				);
-				return this.collectContainerResources(fallback);
+				try {
+					// Inspect operation fails (container ID is out of sync?).
+					const fallback = SUPERVISOR_CONTAINER_NAME_FALLBACK;
+					log.warn(
+						'Supervisor container resources cannot be obtained by container ID. ' +
+							`Using '${fallback}' name instead.`,
+						e.message,
+					);
+					return this.collectContainerResources(fallback);
+				} catch (e) {
+					// Inspect operation fails (using legacy container name?).
```

Comment:
We should make a note somewhere about removing this once we move to 3.x

Reply:
Maybe a GH issue as a reminder?